kindelia_core/benches/bench.rs

```diff
@@ -140,13 +140,13 @@ fn block_loading(c: &mut Criterion) {
   // creates a temporary directory
   let dir = temp_dir();
   // creates the storage with temp dir
-  let storage = persistence::SimpleFileStorage::new(dir.clone());
+  let storage = persistence::SimpleFileStorage::new(dir.clone()).unwrap();
 
   // writes `n` max blocks in disk
   let n = 1000;
   let block = max_block();
   for i in 0..n {
-    storage.write_block(i, block.clone().hashed());
+    let _ = storage.write_block(i, block.clone().hashed());
```

Comment:
Was there a reason for it to be like this?

Reply:
Not really.   It's just that write_block() now returns a Result, so the compiler warns if the Result isn't handled somehow.

These are equivalent in the sense that the compiler is happy with either:

```
let _ = storage.write_block(i, block.clone().hashed());
storage.write_block(i, block.clone().hashed()).unwrap();
```

The first explicitly says "we are ignoring this result/error".  The latter is a bit more implicit.... and also has the potential to panic.    Though maybe that's desirable here, since we aren't propagating errors up in the benchmarks.  (yet?)

So on second thought I will use unwrap() instead.

Reply:
> Though maybe that's desirable here, since we aren't propagating errors up in the benchmarks. 

Yes, I think that in tests and benchmarking it is not necessary to worry about using unwraps. And by using it, it is indicated that this code can fail and that it will fail, if anything goes wrong (and not pass silently without doing anything)